### PR TITLE
termux_step_setup_variables.sh: adding a new variable TERMUX_PYTHON_HOME

### PR DESCRIPTION
**Pull request by Maxython**
This variable allows you to shorten lines that point to python libraries. 
%ci:no-build

### CODE REVIEW

**Review by xtkoba — CHANGES_REQUESTED** [review]
I propose `TERMUX_PYTHON_HOME` as the variable name, after `PYTHONHOME` environment variable (see `python(1)` man page.)

**Comment by xtkoba** [comment]
Also, "value" in the description should be "variable".

**Comment by Maxython** [comment]
Done

**Comment by xtkoba** [comment]
Thanks. Please also fix the subject of the commit message.

**Comment by Maxython** [comment]
Done

**Comment by Maxython** [comment]
Done

File: scripts/build/termux_step_setup_variables.sh [hunk 1]
```diff
@@ -131,6 +131,7 @@ termux_step_setup_variables() {
 	TERMUX_PKG_PYTHON_TARGET_DEPS="" # python modules to be installed via pip3
 	TERMUX_PKG_PYTHON_BUILD_DEPS="" # python modules to be installed via build-pip
 	TERMUX_PKG_PYTHON_COMMON_DEPS="" # python modules to be installed via pip3 or build-pip
+	TERMUX_PYTHON_HOME=$TERMUX_PREFIX/lib/python${TERMUX_PYTHON_VERSION} # location of python libraries
```
Comment (Grimler91) [inline scripts/build/termux_step_setup_variables.sh:134]: Please sort the variables, mv `TERMUX_PYTHON_VERSION` down here as well

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: scripts/build/termux_step_setup_variables.sh (hunk 1 above)
Comment (Maxython) [reply scripts/build/termux_step_setup_variables.sh]: Like so?

Comment (Grimler91) [reply scripts/build/termux_step_setup_variables.sh]: `TERMUX_PYTHON_VERSION` as well:

```
TERMUX_PKG_PYTHON_BUILD_DEPS="" # python modules to be installed via build-pip
TERMUX_PKG_PYTHON_COMMON_DEPS="" # python modules to be installed via pip3 or build-pip
TERMUX_PKG_PYTHON_TARGET_DEPS="" # python modules to be installed via pip3
TERMUX_PYTHON_CROSSENV_PREFIX=$TERMUX_TOPDIR/python-crossenv-prefix # python modules dependency location (only used in non-devices)
TERMUX_PYTHON_HOME=$TERMUX_PREFIX/lib/python${TERMUX_PYTHON_VERSION} # location of python libraries
TERMUX_PYTHON_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python/build.sh; echo $_MAJOR_VERSION) # get the latest version of python
```

Comment (Maxython) [reply scripts/build/termux_step_setup_variables.sh]: Wouldn't that throw an error?

**Comment by xtkoba** [comment]
LGTM. Sorting of variables should be done in another PR if it is needed. Thanks.

**Pull request merged**